src/json.ts

```diff
@@ -9,13 +9,25 @@ export type JSONValue =
   | Array<JSONValue>
   | JSONObject;
 
+/** Like `JSONValue` but deeply immutable */
```

Comment:
The docs can link these if you write `[[JSONValue]]`

Reply:
done